codes/kotlin/chapter_backtracking/n_queens.kt

```diff
@@ -0,0 +1,85 @@
+/**
+ * File: n_queens.kt
+ * Created Time: 2024-01-25
+ * Author: curtishd (1023632660@qq.com)
+ */
+
+package chapter_backtracking
```

Comment:
Looks much cleaner now! Please modify this package name for consistency.

(It is also recommended to update the package names of the array and linked list chapter for better consistency)

Reply:
OK! is Done!